app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="card">
+<div class="d-flex justify-content-between flex-row p-3 mb-1" id="<%= dom_id category %>">
+ <div class="d-flex flex-row">
+    <div class='cate-icon'>
+      <%= image_tag category.icon, height: 50 %>
+    </div> 
```

Comment:
- [ ] [REQUIRED] Great project @Yerimah , However, the icon does not display whenever I try to create a new category. Let's ensure the image displays properly in order to have a great app and meet up with the project requirements.
![Screenshot from 2022-06-02 23-11-09](https://user-images.githubusercontent.com/58692590/171746906-050175f6-d851-4f50-9cfb-d7b90731c52b.png)